src/constants/style.scss

```diff
@@ -7,15 +7,23 @@ $color-white-one: #f9fafb;
 $color-white-two: #f6f5fb;
 $color-dark-one: #242c3d;
 $color-dark-two: #272f41;
-$color-light-gray: #edeff2;
+$color-lighter-gray: #edeff2;
+$color-light-gray: #c3c9d3;
 $color-middle-gray: #a3a6aa;
 $color-dark-gray: #859093;
 $color-darker-gray: #4f5253;
 $color-darkest-gray: #232323;
 $color-black: #111111;
+
+// element colors
 $color-dark-mode: #272f41;
+$color-dark-mode--disabled: #1b202d;
 $color-dark-mode-note: #313949;
 $color-dark-mode-note--hover: #3d4555;
+$menu-icon-background-color--dark: #333948;
+$menu-icon-background-color--light: $color-white;
+$tooltip-background-color--light: #d3daf0;
+$tooltip-background-color--dark: #485064;
```

Comment:
Do we need all those extra colors? We already have many different hues, can't we just re-use them? Would make for a cleaner look overall imo

Reply:
Mhm I just compared the colors with the existing ones and they are different i.e. unique. E.g. `$menu-icon-background-color--dark` is darker than `$color-darker-gray` but visibly lighter than `$color-darkest-gray` (and also a little blue-ish).

Btw, I just moved the constants from `MenuBars.scss` to `styles.scss` bc some were used in multiple files.

Didnt someone refactor the MenuBar colors for better visibility just recently?

Reply:
I just talked to @bitionaire. We need those colors in the future, so we cant replace them but we should try to fit them in our current color scheme. Since they dont fit easily (see example above), this should be handled in a separate PR.